src/models/converters/transactions/tests/summary.rs

```diff
@@ -291,5 +291,44 @@ fn multisig_transaction_to_erc721_transfer_summary() {
 
     let actual = MultisigTransaction::to_transaction_summary(&multisig_tx, &mut mock_info_provider);
 
+    assert_eq!(&expected, actual.unwrap().get(0).unwrap());
+}
+
+#[test]
+fn multisig_transaction_to_ether_transfer_summary() {
+    let multisig_tx = serde_json::from_str::<MultisigTransaction>(crate::json::MULTISIG_TX_ETHER_TRANSFER).unwrap();
+    let safe_info = serde_json::from_str::<SafeInfo>(crate::json::SAFE_WITH_MODULES).unwrap();
+
+    let mut mock_info_provider = MockInfoProvider::new();
+    mock_info_provider
+        .expect_safe_info()
+        .times(1)
+        .return_once(move |_| Ok(safe_info));
+    mock_info_provider
+        .expect_token_info()
+        .times(1) // isErc20 or isErc721 check requires the token to be checked, Question: why do we use MultisigTx.to for fetching the token info?
```

Comment:
Hey @rmeissner while writing the tests for this I ran into something I don't understand. The line is here: https://github.com/gnosis/safe-client-gateway/blob/3448a490b3580539c3da0c3dd65861f98f65db01/src/models/converters/transactions/mod.rs#L52

Reply:
Is the `to` address used as a cache key or something?

Reply:
Under the hood this is used as a cache key (as it should be an address)

Reply:
Should I refactor to make this request only when we've confirmed this is not a `settingsChange` nor `custom` nor `etherTransfer` ? Or could this maybe mess things up?

Reply:
does the type of token transfer checking order matter, is basically my question

Reply:
We should refactor this at some point but not in this issue. I would refactor the whole logic how we check what kind of transfer it is and so on. Can you create an issue for that?

Reply:
Cool, will do 👍